pipeline/src/transformers/addressables/book.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  asText,
+  asTitle,
+  isFilledLinkToDocumentWithData,
+  isNotUndefined,
+} from '@weco/content-pipeline/src/helpers/type-guards';
+import { BookPrismicDocument } from '@weco/content-pipeline/src/types/prismic/books';
+import { ElasticsearchAddressableBook } from '@weco/content-pipeline/src/types/transformed';
+
+export const transformAddressableBook = (
+  document: BookPrismicDocument
+): ElasticsearchAddressableBook => {
+  const { data, id, uid } = document;
+  const primaryImage = data.promo?.[0]?.primary;
+  const description = primaryImage?.caption && asText(primaryImage.caption);
+  const title = asTitle(data.title);
+  const subtitle = data.subtitle ? asText(data.subtitle) : undefined;
+  const titleSubtitle = `${title}${subtitle ? `: ${subtitle}` : ''}`;
+  const contributors = (data.contributors ?? [])
+    .map(c => {
+      return isFilledLinkToDocumentWithData(c.contributor)
+        ? asText(c.contributor?.data.name)
+        : undefined;
+    })
+    .filter(isNotUndefined);
```

Comment:
genuine don't know; could there be more contributors than just the author? Should we only get Authors? (it might not be a real issue)

Reply:
Apparently so https://wellcomecollection.org/books/brains

Reply:
Let's leave it like that and if we get comments we'll amend.

Reply:
I think if there was e.g. an author and an illustrator listed in the contributors, we'd want to display both in the UI. I've made it a string and comma separated them for now